# Homework 1
My work flow changed about half way through the project when I realized it would be much easier ot separate out maize and teosinte first thing before going any further. This made the fang_et_al file much easier to work with. 
From there I was able to transpose, sort, and join with snp_cut which had itself been cut and then sorted. 
 Next was to sort out by chromosome column using a for loop Corrinne helped me write wihch also included sorting for chromosome position
Files were then sorted into four folders: maize.dash, maize.question, teosinte.dash, teosinte.question

##Homework 2
#Part 1
Maize and teosinte were pulled out of the fang file first. Files were then transposed and merged with SNPs file. From there files went through as sort, followed by a loop that allowed me to pull out ascending and descending chromosome files. Descending also went through a replace command. 40 files were generated. 

#Part 2
I wrote for loops which would melt each file and was in the process of adding a graph command, when I realized none of my files had Group columns. They were lost way back in the merge step. Since I had already invested 12+ hours in this assignment I spent a few more looking for a fix but was unable. Some other students said we should be using original data instead but since Fang only had genotypes and groups, and snps only had chromosomes, I was not sure which file to use to graph. Therefor part 1 is complete, but I was unable to finish Part 2. 
